test/e2e/12_commit_signing.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+
+load lib/env
+load lib/gpg
+load lib/install
+load lib/poll
+
+function setup() {
+  setup_env
+  kubectl create namespace "${FLUX_NAMESPACE}"
+
+  generate_ssh_secret
+  install_git_srv
+
+  tmp_gnupghome
+  gpg_key=$(create_gpg_key)
+  create_secret_from_gpg_key "$gpg_key"
+  install_flux_gpg "$gpg_key"
+}
+
+@test "Git sync tag is signed" {
+  # Test that a resource from https://github.com/fluxcd/flux-get-started is deployed
+  # This means the Flux instance _should_ have pushed a signed high-watermark tag
+  poll_until_true 'namespace demo' 'kubectl describe ns/demo'
+
+  # Test that the tag has been signed, this errors if this isn't the case
+  kubectl --namespace "${FLUX_NAMESPACE}" exec -it deployment/flux-gpg \
+    -- sh -c "cd /tmp/flux-gitclone* && git verify-tag flux-sync" >&3
+}
+
+@test "Git commits are signed" {
+  # Assure the resource we are going to lock is deployed
+  poll_until_true 'workload podinfo' 'kubectl -n demo describe deployment/podinfo'
+
+  # Let Flux push a commit
+  fluxctl --k8s-fwd-ns "${FLUX_NAMESPACE}" lock --workload demo:deployment/podinfo >&3
+
+  # Sync right away, this will assure the clone we will look at next is up-to-date
+  fluxctl --k8s-fwd-ns "${FLUX_NAMESPACE}" sync >&3
+
+  # Test that the commit has been signed
+  kubectl --namespace "${FLUX_NAMESPACE}" exec -it deployment/flux-gpg \
```

Comment:
This is ok for now but, as I mentioned on Slack, I am working on a way to access the git server from outside the cluster.